Firmware/communication/ascii_protocol.cpp

```diff
@@ -26,20 +26,36 @@
 
 /* Private variables ---------------------------------------------------------*/
 /* Private function prototypes -----------------------------------------------*/
+
+void setPosition(char * pStr, StreamSink& response_channel, bool use_checksum);
```

Comment:
Our naming convention for functions is snake_case (e.g. `set_position`). To make it clear that these functions belong together as a group you could add a prefix (`cmd_set_position` or similar).

Reply:
Not necessarily, up to you. Oh btw since they are local you could prepend `static`.